src/views/virtualmachines/actions/components/VirtualMachineMigration/hooks/useMigratableStorageClasses.ts

```diff
@@ -1,26 +0,0 @@
-import { V1beta1StorageSpecAccessModesEnum } from '@kubevirt-ui/kubevirt-api/kubevirt';
-import { StorageProfileModelGroupVersionKind } from '@kubevirt-utils/models';
-import { getName } from '@kubevirt-utils/resources/shared';
-import { K8sResourceCommon, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
-
-const useMigratableStorageClasses = (): string[] => {
```

Comment:
removal seems not related to the PR

Reply:
Yes its related becuase MTC migration support all storage classes  instead of `updateVolumesStrategy`